src/Nox.Solution/Models/Domain/Entities/Entity.cs

```diff
@@ -125,20 +125,19 @@ internal bool ApplyDefaults()
 
         if (Relationships is not null)
         {
-#pragma warning disable S3267 // Loops should be simplified with "LINQ" expressions
-            foreach (var relationship in Relationships)
+            var relationships = Relationships
```

Comment:
why are you change this in this PR?

code cleanup needs to be done in its own PR